### PR TITLE
export INotifyOption

### PR DESCRIPTION
**Pull request by oreporan**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Add or edit tests to reflect the change. (Run with `npm test`.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [ ] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

If removing a declaration:
- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: types/notifyjs/index.d.ts [hunk 1]
```diff
@@ -115,4 +115,4 @@ interface INotifyOption {
      */
     requireInteraction?: boolean;
 }
-export = Notify
+export default Notify
```
Comment (NateScarlet) [inline types/notifyjs/index.d.ts]: `export =` is correct but `export default` not.

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default

```shell
> node -p 'var window = {}; require("notifyjs")'
{ [Function: Notify]
  isSupported: [Function],
  needsPermission: true,
  requestPermission: [Function] }
```

```shell
> node -p 'var window = {}; require("notifyjs").default'
undefined
```

**Review by NateScarlet — CHANGES_REQUESTED** [review]

**Review by NateScarlet — CHANGES_REQUESTED** [review]
same as above

**Comment by oreporan** [comment]
@NateScarlet I'm trying to leave the default export `Notify`but also export the `INotifyOptions` interface. How can I do that?

**Review by NateScarlet — APPROVED** [review]

**Comment by NateScarlet** [comment]
Ready to merge

**Pull request merged**